content/en/docs/ambient/architecture/index.md

```diff
@@ -9,15 +9,66 @@ owner: istio/wg-networking-maintainers
 test: n/a
 ---
 
+## Ambient APIs
+
+### Labels
+
+You can use the following labels to enroll your namespace to ambient, enroll your resource to use a waypoint and specify, the capture scope of your waypoint.
+
+|  Name  | Feature Status | Scope | Description |
+| --- | --- | --- | --- | --- |
+| `istio.io/dataplane-mode` | Beta | `Namespace` |  Specifies the data plane mode, valid value: `ambient`. |
+| `istio.io/use-waypoint` | Beta | `Namespace` or `Service` or `Pod` | Enrolls your resource to use a given waypoint, valid values: `{waypoint-name}`, `{namespace}/{waypoint-name}`, or `#none` |
+| `istio.io/waypoint-for` | Alpha | `Gateway` | Specifies the waypoint's capture scope, valid value: `service` or `none` or `workload` or `all`. This label is optional and the default value is `service`. |
```

Comment:
If we want to define a reusable term like 'capture scope' when we need to be a bit more verbose

"Specifies the waypoint's 'capture scope', the class of addressed endpoints it is willing to accept, valid values: `service` or `none` or `workload` or `all`. This label is optional and the default value is `service`. |

or we can avoid trying to define the term and rely on the natural use of 'for' to explain

Specifies what types of endpoints the  waypoint will process traffic for, valid values: `service` or `none` or `workload` or `all`. This label is optional and the default value is `service`. |

Reply:
updated, took your 2nd suggestion :)